docs/standard/delegates-lambdas.md

```diff
@@ -40,9 +40,9 @@ public class Program
 
 In order to streamline the development process, .NET includes a set of delegate types that programmers can reuse and not have to create new types. These are `Func<>`, `Action<>` and `Predicate<>`, and they can be used in various places throughout the .NET APIs without the need to define new delegate types. Of course, there are some differences between the three as you will see in their signatures which mostly have to do with the way they were meant to be used:
 
-* `Action<>` is used when there is a need to perform an action using the arguments of the delegate.
-* `Func<>` is used usually when you have a transformation on hand, that is, you need to transform the arguments of the delegate into a different result. Projections are a prime example of this.
-* `Predicate<>` is used when you need to determine if the argument satisfies the condition of the delegate. It can also be written as a `Func<T, bool>`.
+* `Action<>` is used when there is a need to perform an action using the arguments of the delegate. The method it encapsulates does not return a value.
```

Comment:
I think it's a good idea to add the return values to the definitions here. I'm not 100% sure that "encapsulates" is technically accurate, since the method is assigned to the delegate type. @billwagner Can you comment if "encapsulates" is correct here?

Reply:
@gewarren 
Yes, I agree that we need some more insight into that if it creates confusion. To be honest, I was using msdn docs referring to these particular delegates :).